<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>547</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rooms">
       <item>
        <property name="text">
         <string>121</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>122</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>123</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>124</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>418</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>37</y>
      <width>541</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Action_open"/>
    <addaction name="Action_save"/>
    <addaction name="ActionSave_as"/>
    <addaction name="separator"/>
    <addaction name="Action_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="Action_room"/>
    <addaction name="Action_teacher"/>
    <addaction name="Action_course"/>
    <addaction name="Action_class"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Action_open">
   <property name="text">
    <string>Action_open</string>
   </property>
  </action>
  <action name="Action_save">
   <property name="text">
    <string>Action_save</string>
   </property>
  </action>
  <action name="ActionSave_as">
   <property name="text">
    <string>ActionSave_as</string>
   </property>
  </action>
  <action name="Action_quit">
   <property name="text">
    <string>Action_quit</string>
   </property>
  </action>
  <action name="Action_room">
   <property name="text">
    <string>Action_room</string>
   </property>
  </action>
  <action name="Action_teacher">
   <property name="text">
    <string>Action_teacher</string>
   </property>
  </action>
  <action name="Action_course">
   <property name="text">
    <string>Action_course</string>
   </property>
  </action>
  <action name="Action_class">
   <property name="text">
    <string>Action_class</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
